requirements/common.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
-# by the following command:
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
 #
-#    pip-compile --generate-hashes --output-file=requirements/common.txt requirements/common.in
+#    pip-compile --generate-hashes --output-file='requirements\common.txt' 'requirements\common.in'
```

Comment:
nit: this looks like windows backslashes, can we use unix ones?  you can search and replace in the file

Reply:
It's autogenerated and depends on the OS on which the script ran.

Reply:
I know, but it is unnecessary changes to put in, I run on windows and most of the time remember to search and replace the backslashes.